meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,169 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 2019-02-11
+
+Please note. There has been a  [meeting format change](https://github.com/ipfs/team-mgmt/issues/827#issuecomment-452157617) intended to make our brief sync time more valuable and more easily understood by new/occasional attendees.
+
+**Meeting Lead:** @eingenito
+
+**Attendees:**
+  - @hannahhoward
+  - @jimpick
+  - @aschmahmann
+  - @eingenito
+  - @michaelavila
+  - @stebalien
+  - @bigs
+  - @djdv
+  - @kubuxu
+  - @mgoelzer
+  - @magik6k
+  - @warpfork
+  
+**Recording:** [Recordings are available to the core team only](https://drive.google.com/drive/u/2/folders/1iyFnbY4RRsfIcS_b6M9wMEjbuZceSw9z)
+
+
+## Agenda
+- Review Announcements for the whole team
+- Review blockers and needs from other team members (not general updates)
+  - Are you blocked waiting for input?
+  - PRs for review? Remember the [project board](https://github.com/orgs/ipfs/projects/1)
+- Review progress on major initiatives
+  - Find or create the section for the IPFS initiative(s) you worked on.
+  - Summarize the initiative briefly in notes to give a little context.
+  - Summarize your progress
+  - Summarize your next steps.
+- Review other Questions/Announcements/Notes
+
+## Current IPFS Initiatives
```

Comment:
Perhaps rename to "Current Go Core Initiatives" as these are representative of what this team is focusing on